sufia-models/app/jobs/create_derivatives_job.rb

```diff
@@ -8,7 +8,11 @@ def run
     if generic_file.video?
       return unless Sufia.config.enable_ffmpeg
     end
-    generic_file.create_derivatives
+    status = Timeout::timeout(Sufia.config.derivatives_timeout) do
```

Comment:
Wait, doesn't hydra-derivatives already provide this?  Like, can't you just set:

```ruby
Hydra::Derivatives::Image::Processor.timeout = Sufia.config.derivatives_timeout
````

?